### PR TITLE
feature/uf 1998 support subscriptions

### PR DESCRIPTION
**Pull request by istyf**
I believe that this is TSTTCPW to be able to push notifications for entity creations and attribute updates...

### CODE REVIEW

File: internal/pkg/application/subscriptions/notifier_test.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+func notificationEntityID(idx int, is *is.I, entityID string) ValidatorFunc {
+	return func(w http.ResponseWriter, r *http.Request, b []byte) {
+		n := subscriptions.Notification{}
+		json.Unmarshal(b, &n)
+
+		is.Equal(entityID, n.Data[idx].ID()) // entity id should match
+	}
+}
+
+/*func requestBody(is *is.I, body string) ValidatorFunc {
```
Comment (shestyf) [inline internal/pkg/application/subscriptions/notifier_test.go:73]: remove this if it's not longer in use or useful

**Review by shestyf — CHANGES_REQUESTED** [review]
remove the commented out function in notifier_test.go

File: internal/pkg/application/subscriptions/notifier_test.go (hunk 1 above)
Comment (istyf) [reply internal/pkg/application/subscriptions/notifier_test.go]: fixed

**Pull request merged**